### PR TITLE
Minimal design & wording changes

### PR DESCRIPTION
**Pull request by filippo-fonseca**

### CODE REVIEW

**Comment by alii** [comment]
I think while we are at it, should we just abstract the titles and subtitles out to individual components? Having them in global style kind of defeats the purpose of individual components – that's my fault, I wrote them like that initially 😅

What do you think?

**Comment by alii** [comment]
Update: have merged the prettier config from #25 into main, so we can totally remove `.prettierc.yaml`. As for the components, would you be down to abstract those out? Then I think this will be in a mergeable state.

File: .prettierrc.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+singleQuote: false
+semi: true
+jsxSingleQuote: false
+tabWidth: 4
+useTabs: false
+bracketSpacing: true
+arrowParens: avoid
+trailingComma: es5
+proseWrap: preserve
```
Comment (alii) [inline .prettierrc.yaml:9]: We can remove this file

File: src/components/GlobalStyle.ts [hunk 2]
```diff
@@ -66,6 +80,7 @@ export const GlobalStyle = createGlobalStyle`
       
       input {
         margin-top: 5px;
+        outline: none;
```
Comment (alii) [inline src/components/GlobalStyle.ts:83]: Todo: abstract to individual components(?)

File: src/pages/index.tsx [hunk 3]
```diff
@@ -29,17 +29,25 @@ export default function Index() {
...
-        <span className="subtitle">Fetching the latest themes</span>
+        <span className="subtitle">🧪 Hold up! Fetching the latest themes...</span>
       </div>
     );
 
   return (
     <div className="app">
       <h1 className="title">Linear Style</h1>
-      <span className="subtitle">Click on a theme to copy it.</span>
+      <span className="subtitle">Click on a theme to copy it. Then, paste into your Linear.</span>
```
Comment (alii) [inline src/pages/index.tsx:39]: This subtitle was fine text wise as it was before, I think this is too _waffly_ 😝

**Review by alii — CHANGES_REQUESTED** [review]
Here's a small review with the things we can do :)

**Comment by filippo-fonseca** [comment]
I will get on it @alii. Thanks for the feedback.

**Comment by alii** [comment]
Awesome stuff, thanks for your contributions!

**Comment by filippo-fonseca** [comment]
Done! @alii

**Comment by alii** [comment]
Nice one man, thank you so much! ❤️

**Pull request merged**